.github/workflows/cve-scanning.yml

```diff
@@ -25,9 +25,9 @@ jobs:
       matrix:
         node-version: [20]
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@1a4442cacd436585916779262731d5b162bc6ec7 # v3.8.2
```

Comment:
Might be worth bumping to 4.0.2 / 60edb5dd545a775178f52524783378180af0d1f8 (not tested here but used in the pacakge workflow) to be up to date.